langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -99,6 +112,14 @@ public isolated function sum(int... ns) returns int = @java:Method {
 
 # Returns the maximum of one or more int values.
 #
+# ```ballerina
+# // Retrieving the highest value out of the list of integers
```

Comment:
I think we do not need this comment.

Reply:
Yes, but we need to add several examples. So we need this